src/remixAppManager.js

```diff
@@ -202,6 +202,22 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://zokrates.blockchain-it.hr/zokrates.svg',
       location: 'sidePanel'
     }
+    const quorum = {
+      name: 'quorum',
+      displayName: 'Quorum Support',
+      description: 'Enables support for deploying and interacting with private contracts on a Quorum network.',
```

Comment:
I would recommend something more direct : `Deploy and interact with private contracts on a Quorum network.`